src/methods/defining_methods/hard/q3/Polygo.java

```diff
@@ -0,0 +1,31 @@
+package methods.defining_methods.hard.q3;
+
+/*
+Write a Java Program that asks the user for the number of sides on a polygon, then determines the sum of the measure of
+the interior angles of that polygon. Remember that the formula to determine the sum of interior angles is (n-2)*180,
+where n is the number of sides. Create a method called "innerAngles(int size)" that takes the number of sides and
```

Comment:
I think that because this is a hard question in one of the later categories, we can add in that the method should check that the input is greater than 2, and return -1 otherwise